data/wanderer/wanderers start.txt

```diff
@@ -768,6 +769,14 @@ mission "Wanderers: Unfettered Diplomacy 1B"
 		personality timid escort
 		ship "Deep River (Jump, Unarmed)" "Ayi'i Ki'ak'var"
 
+fleet "Unfettered Tribute (Wanderer)"
+	government "Hai (Unfettered) (Wanderer Tribute)"
+	names "hai"
+	cargo 3
+	variant
+		"Water Bug (Jump)"
+
+# TODO: this fleet is considered deprecated, but preserved for old save files with references to it.
```

Comment:
I'd move this fleet in the hai fleet.txt folder if you want to keep it

Reply:
If someone has a good use for it elsewhere, they can move it as appropriate, but since it's not used anywhere else, and only exists to preserve compatibility with where it was previously used (entirely in Wanderer stuff), I think it should stay here.

Reply:
yeah it won't ever be used anywhere else since water bugs are no longer a UHai thing, and any future tributes would use the Hai (friendly uHai)